### DISCUSSION TITLE
Welcome to support Discussions!

### DISCUSSION THREAD

**Discussion by leo816** [bisq-network/support#891] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Theoldgits** [comment]
I only set up a Bisq trading account yesterday and already both are 2 failed transactions.

I have encountered the following error twice and I don't understand why the multisig transaction is missing:

The trade protocol encountered problems.

The deposit transaction (the 2-of-2 multisig transaction) is missing.

Trade state: Deposit transaction has been published

Bisq version: 1.9.10


I keep reading that deleting the SPV file and resyncing should fix the error. I tried this on both failed transactions and it does not work.

How can I solve this problem because I cannot trade with a platform that keeps causing errors. No other platform has done this before.

**Comment by pazza83** [comment]
I would recommend you reaching out to get live support on Bisq Matrix:

https://bisq.wiki/Matrix_bisq.chat

Failed trades can happen for a number of reasons if you chat to someone on the support team they will be able to find out what is causing it an offer you the best steps to take.

Easier to fix them in Matrix as it allows back and forth chat and better privacy as some transaction IDs will be likely needed to find out what happened.

https://bisq.wiki/User_support
